model_compression_toolkit/exporter/model_exporter/pytorch/base_pytorch_exporter.py

```diff
@@ -73,8 +73,22 @@ def _set_quantized_weights_in_wrapper(layer:PytorchQuantizationWrapper):
     for name in layer.weights_quantizers.keys():
         quantized_weight = torch.nn.Parameter(layer.get_quantized_weights()[name]).detach()
         linear_layer = getattr(layer, LAYER)
-        delattr(linear_layer, name)
-        setattr(linear_layer, name, torch.nn.Parameter(quantized_weight))
+
+        # If the name is a string, we assume it's a named attribute of the linear layer
+        if isinstance(name, str):
+            # Remove the existing attribute from the linear layer
+            delattr(linear_layer, name)
+            # Replace it with the quantized version as a new parameter
+            setattr(linear_layer, name, torch.nn.Parameter(quantized_weight))
+        else:
+            # If the name is not a string, it must be an integer representing a positional weight
+            assert isinstance(name, int)
+            attr_name = f'{QUANTIZED_POSITIONAL_WEIGHT}_{name}'
+            if hasattr(layer, attr_name):
+                delattr(layer, attr_name)
+
+            # Add the quantized weight as a new attribute directly on the parent layer
```

Comment:
explain we do that so the wrapper forward will work. that's why we select that specific attribute name.